fastestimator/backend/random_mix_patch.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2019 The FastEstimator Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+from typing import List, TypeVar, Union
+
+import tensorflow as tf
+import torch
+
+Tensor = TypeVar('Tensor', tf.Tensor, torch.Tensor)
+
+
+def random_mix_patch(tensor: Tensor, lam: Tensor, uniform_sample: Tensor) -> Tensor:
+    """Randomly cut the patches from input images.
+
+    Args:
+        tensor: The input value.
+        lam: Generated mixed sample.
+        uniform_sample: Sample drawn from the uniform distribution
+
+    Returns:
+        The X and Y coordinates of the patch along with width and height
+
+    Raises:
+        ValueError: If `tensor` is an unacceptable data type.
+    """
+    if tf.is_tensor(tensor):
```

Comment:
I think this class is kind of missing the point of our backend folder. it ought to be possible to implement this logic in Op without using if branching by using backend calls for maximum and round